migration/v0.16.x-v0.17.x/migrate-dex-additional-static-clients.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+
+set -euo pipefail
+
+: "${CK8S_CONFIG_PATH:?Missing CK8S_CONFIG_PATH}"
+
+here="$(dirname "$(readlink -f "${BASH_SOURCE[0]}")")"
+
+secret="${CK8S_CONFIG_PATH}/secrets.yaml"
+secret_tmp="${CK8S_CONFIG_PATH}/secrets_tmp.yaml"
+sops_conf="${CK8S_CONFIG_PATH}/.sops.yaml"
+
+update_script=$(mktemp -p /tmp update-script.XXXXXXXXXX.yaml)
+cp "${here}/dex-update-script.yaml" "${update_script}"
+
+# read all dex's static clients from ck8s cluster
+static_clients_yaml=$(./bin/ck8s ops kubectl sc get secret dex --namespace dex --output json | jq --raw-output '.data."config.yaml"' | base64 --decode | yq read - staticClients)
```

Comment:
I don't think it matters, I mainly wanted to highlight that you can use `jsonpath` instead of piping to jq. So you can change it if you want or just ignore this

```suggestion
static_clients_yaml=$(./bin/ck8s ops kubectl sc get secret dex --namespace dex --output jsonpath='{.data.config\.yaml}' | base64 --decode | yq read - staticClients)
```

Reply:
Cool, but somehow it's not printing any output. It's probably the `config.yaml`, because it prints fine for `{.data}`. I'll see if I can get that working.